src/schema.ts

```diff
@@ -1381,6 +1391,7 @@ export const typeDefs = gql`
       includeResources: Boolean
     ): SharedFolder!
     allFolderResources(size: Int): [FolderResource!]!
+    myNdlaUser: MyNdlaPersonalData
```

Comment:
La til et nytt endepunkt som kan returnerere undefined. Kunne kanskje brukt det gamle. Innspill?

Reply:
Tja, er det noen grunn til at vi har behov for at brukerdata ikke eksisterer?

Reply:
Kun det at om du ikkje er logga på så vil endepunktet returnere undefined og ikkje kaste exeption.

Reply:
Godt poeng. Jeg kan fint leve med at dette er tilfellet overalt isåfall

Reply:
Då kan eg endre slik at vi kun har ett endepunkt og ikkje to.